### PR TITLE
Create security-guidelines.md

### PR DESCRIPTION
**Pull request by monicazng**

### CODE REVIEW

File: src/settling-in/security-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+> In much the same way we've decentralized the organization and applications, we've done the same for security.
+>
+> It means the vast majority of the attack surface is you, the people that contribute. Furthermore, the controls and information a security specialist has in terms of making decisions and monitoring how things are run is constrained. It also means education takes a forward step in importance across the organization, and personal responsibility of quality security practices become paramount. It is up the people to understand potential threats, take preventative measures, and report any issues they come across to the security team to protect the organization as a whole.
+> – @petty
+
+## Ask for help
+
+Above all else, never be afraid to ask for help, ask questions, or report security concerns. Until we have an official infrastructure for support tickets, drop by #security-helpdesk to get help.
+
```
Comment (corpetty) [inline src/settling-in/security-guidelines.md:15]: also include #security-internal for anything that is more private, or #security for more broad, public questions.

Comment (corpetty) [reply src/settling-in/security-guidelines.md:15]: this should stay updated as we transition to various places. I'll continue to add resources for you to add.

Comment (corpetty) [reply src/settling-in/security-guidelines.md:15]: the security@status.im email is also available.

File: src/settling-in/security-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+## Hardware
+
+Check out this [list](https://notes.status.im/zkr20eWFTwGD92xixGX2CA) of essential hardware.
+
+## Password manager
+
+Our password manager is Bitwarden; everyone is free to use it to store and secure passwords. You can request an invite from @terry.
+
+## Security team
+
```
Comment (corpetty) [inline src/settling-in/security-guidelines.md:33]: probably link [security-internal](github.com/status-im/security-internal) and [status-security](github.com/status-im/status-security) resources, which hold internal and public security information, respectively.

Comment (corpetty) [reply src/settling-in/security-guidelines.md:33]: thoughts @status-im/security-internal on anything else to add or change?

File: src/settling-in/security-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+Phishing attacks are the most common: discord bots, telegram messages, and email, amongst other channels. Take this [phishing test](https://phishingquiz.withgoogle.com/) to see how savvy you are when it comes to detecting phishing.
+
+## Hardware
+
+Check out this [list](https://notes.status.im/zkr20eWFTwGD92xixGX2CA) of essential hardware.
+
+## Password manager
+
+Our password manager is Bitwarden; everyone is free to use it to store and secure passwords. You can request an invite from @terry.
```
Comment (corpetty) [inline src/settling-in/security-guidelines.md:30]: all organizational passwords should be kept and shared here. So if you plan to store a Status related password, or get access to platforms that require username/passwords/2fa, then it should all be shared here. 

By signing up with the organization, you get the premium features, and can sign up with any email you like. That way if you ever leave Status you can take your password manager secrets with you, and you just lose access to Status related ones.

File: src/settling-in/security-guidelines.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+This open-source security [checklist](https://securitycheckli.st/) helps you improve your online security and privacy in a clear, easy-to-follow list format. You can check off completed items as you go along.
+
+Fancy giving your security a spring clean? Check out [this](https://winter.mycrypto.com/) crypto advent calendar with daily bite-sized security tips.
+
+Phishing attacks are the most common: discord bots, telegram messages, and email, amongst other channels. Take this [phishing test](https://phishingquiz.withgoogle.com/) to see how savvy you are when it comes to detecting phishing.
+
+## Hardware
+
+Check out this [list](https://notes.status.im/zkr20eWFTwGD92xixGX2CA) of essential hardware.
+
```
Comment (corpetty) [inline src/settling-in/security-guidelines.md:27]: hardware security keys (yubikeys) can be expensed by a CC, and should be used for Github/Gsuite/Bitwarden.

Comment (corpetty) [reply src/settling-in/security-guidelines.md:27]: something regarding hardware wallets could also go here, maybe link the learn-up video here as well. 

https://www.youtube.com/watch?v=9nzqntuFR44

**Review by corpetty — CHANGES_REQUESTED** [review]

File: src/settling-in/security-guidelines.md (hunk 4 above)
Comment (monicazng) [reply src/settling-in/security-guidelines.md:27]: This is great, adding it!

**Review by serhanwbahar — APPROVED** [review]

**Review by OxFred — APPROVED** [review]

**Pull request merged**